src/coreclr/vm/exstate.h

```diff
@@ -202,7 +202,11 @@ class ThreadExceptionState
             ThrowCallbackType& tct,
             Frame *pStartFrame);
 
-    friend EXCEPTION_HANDLER_IMPL(COMPlusFrameHandler);
+    friend EXCEPTION_DISPOSITION __cdecl
```

Comment:
Use `_EXCEPTION_HANDLER_DECL`?

Reply:
I'll revert this line back to `EXCEPTION_HANDLER_IMPL` (needs the version without `extern "C"` prefix) and fix the other occurrence in the file to use it too.

Reply:
Ah, there's the underscore variant too... 👍🏿